frontend/src/scenes/settings/project/SessionRecordingSettings.tsx

```diff
@@ -346,165 +347,159 @@ export function ReplayCostControl(): JSX.Element | null {
     const { updateCurrentTeam } = useActions(teamLogic)
     const { currentTeam } = useValues(teamLogic)
     const { hasAvailableFeature } = useValues(userLogic)
-    const { featureFlags } = useValues(enabledFlagsLogic)
-
-    // some organisations have access to this by virtue of being in a flag
-    // other orgs have access by virtue of being on the correct plan
-    // having the flag enabled overrides the plan feature check
-    const flagIsEnabled = featureFlags[FEATURE_FLAGS.SESSION_RECORDING_SAMPLING]
```

Comment:
yep, we're only controlling it on plan now so should be 👍 

(i guess technically we should check who the flag is enabled for, whether they have set controls, and whether they only have access because of the flag - but they'll tell us soon enough so :yolo:)

Reply:
I guess we can remove the flag from the constants file (and anywhere else it is referenced too)

Reply:
occurred to me that waiting for the sun to reach america was silly so I cleaned it up 😊